subsys/bluetooth/controller/ll_sw/ull_internal.h

```diff
@@ -8,11 +8,15 @@
  *  User CPR Interval
  */
 #if !defined(CONFIG_BT_CTLR_USER_CPR_INTERVAL_MIN)
+#if defined(CONFIG_BT_CTLR_CONN_INTERVAL_LOW_LATENCY)
+#define CONN_INTERVAL_MIN(x) (0U)
```

Comment:
Is `0` right here? Shouldn't it be `1`?

Reply:
Hmm, `0` means `500us` sounds weird to me :D If `ms` isn't possible to use, then perhaps these macros should be in `us` instead? It sounds weird that an interval of 0 is not in fact 0.